renders data

Euniceem opened the pull request:
.env file is hidden and I got the data to render on the frontend.

Euniceem committed:
done with models

Euniceem committed:
got data to show up

Euniceem committed:
got data to render

Euniceem committed:
fixed code

Euniceem committed:
Delete .env

Euniceem committed:
hide env

Euniceem committed:
Delete .env

Euniceem committed:
fixed code for rendering kanbanboard

Euniceem committed:
made changes on models for routes

jaywon commented on .dockerignore:
```diff
@@ -0,0 +1,3 @@
+Dockerfile
```
This is great but just FYI you could use a little regex to condense this down to one line which is something I do often:

`[Dd]ocker*`

This will match upper *or* lower case `D` followed by all lower case `ocker` and any number of characters after that with the `*`.

jaywon commented on database/index.js:
```diff
@@ -1,3 +1,4 @@
-const environment = process.env.NODE_ENV || 'development'
+const environment = process.env.NODE_ENV || 'development' //remove
 module.exports = require('knex')(config[environment]);
+// module.exports = require('knex')(config);
```
In general you don't want to leave comments in when doing PRs, if anything it's more preferred to leave a `TODO` statement than commented out code:

`TODO: Refactor knex config to not use environment`

IDE's often have search for `TODO` keyword capabilities

jaywon commented on knexfile.js:
```diff
@@ -1,14 +1,13 @@
-      user: 'euniceem1007',
-      password: 'password',
-      database: 'react-kanban'
+      user: process.env.POSTGRES_USER,
```
This looks good with no sensitive information committed. Ask an instructor and we can show you how to condense it down to one single object 😃

jaywon commented on server/server.js:
```diff
@@ -4,13 +4,14 @@ const bodyParser = require('body-parser');
+const SESSION_SECRET = process.env.SESSION_SECRET
 // app.use(session({
-//   store: new redis({ url: 'redis://redis-server:6379', logErrors: true }),
+//   store: new redis({ url: 'redis://redis-server:REDIS_HOST_PORT', logErrors: true }),
```
This is good but we want to configure both the Redis URL _and_ port so something like this:

```
store: new redis({ url: `${REDIS_URL}:${REDIS_PORT}`, logErrors: true })
```

jaywon commented on src/App.js:
```diff
@@ -1,25 +1,57 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+// import { connect } from 'react-redux';
```
See previous comment about comments 👍

jaywon commented on src/App.js:
```diff
@@ -1,25 +1,57 @@
-        </header>
+        <KanbanTitle title={this.state.pageTitle} />
+
+        <KanbanBoard data={this.cards} />
```
A better attribute than `data` in this instance might be `cards`?

jaywon commented on src/components/KanbanBoard/KanbanBoard.jsx:
```diff
@@ -0,0 +1,36 @@
+    super(props)
+  }
+  render() {
+    const data = this.props.data;
```
If you changed the attribute to `cards` in App.js you would need to update this to `this.props.cards` 🌮

jaywon commented on src/components/KanbanCard/KanbanCard.jsx:
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const KanbanCard = (props) => {
+  const { title, priorityName, created_byFirstName, assigned_byFirstName } = props;
```
In general for variable naming I would recommend sticking w/ all camel case and not a mix of underscores and casing:

`createdByFirstName`

`assignedByFirstName`

jaywon commented on src/components/KanbanColumn/KanbanColumn.jsx:
```diff
@@ -0,0 +1,69 @@
+
+
+    const inProgress = cardList.filter((card) => {
+      return card.status.includes('In-Progress')
```
Is there a reason for using `includes` vs a explicit match? Just curious.

jaywon reviewed (changes requested):
Please see feedback and let us know if you have any questions 🎉

jaywon commented:
Also, one other thing. Can you make your commit messages a little bit more descriptive based on the change the commit introduces. Rather than `fixed code` something like `Add Redis environment configuration variables to Express server`

Euniceem committed:
seeds working

Euniceem committed:
got rid of unnecessary comments and condensed my knexfile into one object

Euniceem committed:
Untrack files in .gitignore

Euniceem committed:
Merge pull request #3 from Euniceem/testdev

got rid of comments and condensed my knexfile

Euniceem committed:
got rid of slashes

Euniceem committed:
condensed knexfile

Euniceem committed:
got get route working with the frontend and renders data onto browser

Euniceem committed:
merged with test dev that was reviewed in pr

Euniceem committed:
:wq
:wq

'test'
ss
aerge branch 'routes' of github.com:Euniceem/react-kanban into routes

Euniceem committed:
fixed knexfile

Euniceem committed:
create more cards

Euniceem committed:
changed filename and added more cards in cards seed

Euniceem committed:
erased console logs

Euniceem committed:
changed file name

Euniceem committed:
got routes working

Euniceem committed:
working on implementing add and edit to the frontend

Euniceem committed:
deleted console logs and shortened code

Euniceem committed:
Merge branch 'testroute' into dev

Euniceem committed:
ok by pr
